### PR TITLE
max_frame_size option hardcoded to 8 MBytes. Related to issue #612

### PR DESCRIPTION
**Pull request by cybernetlab**

### CODE REVIEW

**Comment by essen** [comment]
Sounds good but it seems like there's a TODO left?

**Comment by cybernetlab** [comment]
Why do you think so? As I rememeber this PR is ready to merge

File: test/ws_SUITE_data/ws_max_frame_size.erl [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+-module(ws_max_frame_size).
+
+-export([init/2]).
+-export([websocket_init/1]).
+-export([websocket_handle/2]).
+-export([websocket_info/2]).
+
+init(Req, Opts) ->
+	%% TODO: Set max_frame_size to 8 here (or in websocket_init)
```
Comment (essen) [inline test/ws_SUITE_data/ws_max_frame_size.erl]: This todo.

**Comment by cybernetlab** [comment]
Ah, I remember the reason of this TODO. it's regarding these [my](https://github.com/ninenines/cowboy/issues/612#issuecomment-275259829) and [your](https://github.com/ninenines/cowboy/issues/612#issuecomment-275262950) comments. Are your changes already in master?

**Comment by essen** [comment]
Ah right. Yes, these changes are now in master. :-)

**Comment by essen** [comment]
Done via another PR and more changes from me. Thanks!

**Pull request closed without merging**